modules/importer/oggdude/importers/item-attachments.js

```diff
@@ -46,21 +46,24 @@ export default class ItemAttachments {
             metadata: {
               tags: [],
             },
-          };
+          };          
```

Comment:
@obrenckle can you please strip whitespace from here and line 56? You may want to configure your IDE to do this automatically.